layouts/partials/tw/nav.html

```diff
@@ -1,137 +1,268 @@
-        {{ range site.Menus.main }}
-          <a class="site-nav:item" href="{{ .URL }}" data-ga-category="item"
-            >{{ .Name }}</a
-          >
-        {{ end }}
+  <h1
+    class="max-sm:w-44 sm:flex sm:flex-auto sm:items-stretch sm:justify-center"
+  >
+    <a
+      class="h-max flex-1 sm:mx-auto sm:max-w-xs md:max-w-sm lg:max-w-md"
+      href="/"
+      data-ga-category="logo"
+    >
+      <svg class="block h-8 w-full sm:h-14 md:h-16 lg:h-20">
+        <title>Home</title>
+        <desc>Spotlight PA logo</desc>
+        {{ partial "helper/svg.html" "banner" }}
+      </svg>
+    </a>
+  </h1>
+  <a
+    data-sf-campaign="701Dn000000YgogIAC"
+    data-ga-category="donate"
+    class="group relative inline-flex items-center justify-around overflow-hidden rounded-md bg-gradient-to-r from-red-7 to-red p-2 hover:from-blue-7 hover:to-blue sm:flex sm:w-32 sm:justify-center lg:w-52 lg:px-7 lg:py-4"
+    href="/donate"
+  >
+    <span
+      class="absolute inset-0 z-0 bg-red [clip-path:_polygon(0_0,_0%_100%,_15%_50%)] group-hover:bg-blue"
+    ></span>
+    <span
+      class="flex-0 relative inline-flex flex-wrap items-center text-xs font-bold uppercase leading-none tracking-wide text-white lg:text-lg lg:leading-none"
+    >
+      <svg
+        class="mr-1 inline-block h-4 w-4 fill-current lg:h-5 lg:w-5"
+        aria-hidden="true"
+      >
+        {{ partial "helper/svg.html" "heart" }}
+      </svg>
+      Support<span class="hidden sm:inline">&nbsp;Spotlight PA</span>
+    </span>
+  </a>
+  <p
+    class="flex-0 w-full py-3 text-center text-xs font-extrabold leading-none tracking-wide lg:text-sm xl:text-base"
+  >
+    Journalism that gets results for Pennsylvania
+  </p>
+  <ul
+    class="-order-1 hidden text-xs font-extrabold leading-tight tracking-wide sm:block sm:w-32 lg:w-52 lg:text-sm xl:text-base"
+  >
+    <li>Independent</li>
+    <li>Nonpartisan</li>
+    <li>Nonprofit</li>
+  </ul>
+</div>
+<nav
+  x-data="{ toggle: false }"
+  class="
+    relative flex h-11 w-full items-stretch justify-between border-b border-t bg-white font-sans md:justify-center
+  "
+  :class="toggle ? 'px-5 sm:px-3' : 'px-3'"
+>
+  <a
+    href="/search/"
+    class="z-20 flex flex-[0_1_70px] flex-row-reverse items-center justify-between gap-1 text-sm font-bold leading-none tracking-wide"
+    data-ga-category="search"
+    :data-ga-action="'modal:search:open'"
+    x-data
+    @click.prevent="$dispatch('modal:search:open')"
+  >
+    <svg class="h-[14px] w-[14px]" aria-label="Search">
+      {{ partial "helper/svg.html" "search" }}
+    </svg>
+    Search
+  </a>
+
+  <button
+    class="
+      no-js:hidden group z-20 order-1 flex flex-[0_1_70px] flex-row items-center gap-1 self-stretch text-sm font-bold leading-none tracking-wide
+    "
+    :class="toggle ? 'sm:mr-3 sm:-ml-3' : 'mx-0'"
+    type="button"
+    x-ref="button"
+    aria-label="Site Menu"
+    :aria-expanded="toggle"
+    :aria-controls="$id('dropdown-button')"
```

Comment:
I think this doesn't work because it doesn't have a corresponding `x-id="['dropdown-button']"` on the thing it opens. See https://alpinejs.dev/directives/id and https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-controls